### PR TITLE
Bug 1101387 - [Build] New makefile target to create the "gaia.zip"

### PR DESCRIPTION
**Pull request by cctuan**
https://bugzilla.mozilla.org/show_bug.cgi?id=1101387

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fdc5944befacc7c72aaf892454b567ef7f5d1bf9

File: Makefile [hunk 1]
```diff
@@ -704,6 +710,12 @@ modules.tar: gaia_node_modules.revision
 	-$(DOWNLOAD_CMD) https://github.com/mozilla-b2g/gaia-node-modules/tarball/$(NODE_MODULES_REV) &&\
 	mv $(NODE_MODULES_REV) "$(NODE_MODULES_SRC)"
 
+$(GAIA_ZIP_TARGET): $(DEFAULT_KEYBOAD_SYMBOLS_FONT) $(DEFAULT_GAIA_ICONS_FONT)
+	@mkdir -p tmp/gaia tmp/gonk/system/fonts/hidden && cp -r $(PROFILE_FOLDER) tmp/gaia && \
+	 cp $(DEFAULT_GAIA_ICONS_FONT) tmp/gonk/system/fonts/hidden && \
+	 cp $(DEFAULT_KEYBOAD_SYMBOLS_FONT) tmp/gonk/system/fonts/hidden
+	@zip -r $(GAIA_ZIP_TARGET) tmp/* && rm -r tmp
```
Comment (askeing) [inline Makefile]: `cd tmp/ && zip -r -9 -u ../gaia.zip . && cd ../ && rm -r tmp/`

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=20a4a535666f31587434186c1095adc1385f868b

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2737e8d714e53af8b18a351874a7c90992fcc2e1

**Pull request merged**